src/store/adjustVault.ts

```diff
@@ -266,6 +266,13 @@ export const adjustVaultErrorsAtom = atom(get => {
     collateralError = 'Insufficient funds.';
   }
 
+  if (
+    collateralAction === CollateralAction.Withdraw &&
+    AmountMath.isEmpty(newLocked)
```

Comment:
why does empty newLocked imply excess? wouldn't it also be zero if you withdraw exactly all of it?

Reply:
The contract won't let you adjust a vault to have 0 collateral, so effectively the "available collateral" to withdraw is 1 less than the amount locked. This was succinct, but we could probably say something like "This would empty the vault completely. Close the vault [with a link to the close action] instead?"

Reply:
Makes sense. Consider documenting with comment

Reply:
Done